debugging/helper-cleanout-server.js

```diff
@@ -109,19 +116,33 @@ exports.run = function (extra, args, cleanout) {
       }
     }
 
+    const t2 = new Date();
+    print("INFO " + (t2));
```

Comment:
This print seems to be a bit lost.
What do we want to see here? RunTime? Then t1 Is missing I guess.
Also I would like to have `+ task +` in there to keep the scope.

Reply:
This is a "progress" bar

Reply:
In order to see that the script is not stuck